docs/data/material/components/init-color-scheme-script/init-color-scheme-script.md

```diff
@@ -0,0 +1,140 @@
+---
+productId: material-ui
+title: InitColorSchemeScript component
+components: InitColorSchemeScript
+githubSource: packages/mui-material/src/InitColorSchemeScript
+---
+
+# InitColorSchemeScript
+
+<p class="description">InitColorSchemeScript component removes the dark mode flicker for server-side rendering application.</p>
+
+## Introduction
+
+`InitColorSchemeScript` component is used to remove the dark mode flicker for server-side rendering (SSR) applications.
+It is a script that runs before React to attach an attribute based on user preference.
+
+It is recommended to use this component whenever:
```

Comment:
passive voice –> "you"-centered action

```suggestion
You should implement this component whenever:
```

Reply:
IIUC, you should use these when _both_ of these statements are true, right? (You wouldn't need it if only one of these statements were true.) In that case, I'd suggest rewriting it as a single sentence:

```suggestion
For the best user experience, you should implement this component in any server-rendered Material UI app that supports both light and dark modes.
```

Reply:
> you should use these when both of these statements are true, right

Yes, that's the message I want to convey.